dev_codeforpoznan_pl_v3.tf

```diff
@@ -60,7 +60,7 @@ module dev_codeforpoznan_pl_v3_ssl_certificate {
 module dev_codeforpoznan_pl_v3_frontend_assets {
   source = "./frontend_assets"
 
-  name      = "codeforpoznan.pl_v3"
+  name      = "dev_codeforpoznan_pl_v3"
```

Comment:
As I understand, that change also changes policies attached to user dedicated to uploading data to this location. I see, that there are resources in that location - does it mean that you uploaded them yourself directly after running that terraform script? @arturtamborski ?

Reply:
yes, the policies were recreated as well. I have uploaded them there myself. (To be super precise - i moved them from previous key `codeforpoznan.pl_v3` where they were already inserted by Travis some time ago).